app/test/socobo-inventory/socobo-inventory-create.html

```diff
@@ -66,7 +67,9 @@
       });
 
       test('create button is hidden when in edit mode', function(done) {
-        dialog.show({});
+        var item = {measurement: {value: 'pcs'}, desc: 'test item', info: 'test info'};
```

Comment:
I think we should use The object, because
- measurement.__firebaseKey__ points to The Position inside The list and we can simplify again The _resetItem function
- we have already rewrite all to use The object